filter_bbox.json

```diff
@@ -39,7 +39,7 @@
         },
         {
             "name": "extent",
-            "description": "A bounding box, which may include a vertical axis (see `base` and `height`).",
+            "description": "A bounding box, which may include a vertical axis (see `base` and `height`).\n\nIf the bounding box is not provided in the coordinate reference system (CRS) of the data cube, the bounding box is reprojected to the CRS of the spatial data cube dimensions.",
```

Comment:
This clarification  also applies to `spatial_extent` in `load_collection`, but it's probably a bit messier to do there as multiple data types are allowed

Reply:
Added.